runtime/vm/ContinuationHelpers.cpp

```diff
@@ -570,17 +586,34 @@ releaseVThreadInspector(J9VMThread *currentThread, jobject thread)
 	MM_ObjectAccessBarrierAPI objectAccessBarrier = MM_ObjectAccessBarrierAPI(currentThread);
 	j9object_t threadObj = J9_JNI_UNWRAP_REFERENCE(thread);
 	I_64 vthreadInspectorCount = J9OBJECT_I64_LOAD(currentThread, threadObj, vm->virtualThreadInspectorCountOffset);
-	/* vthreadInspectorCount must be greater than 0 before decrement. */
-	Assert_VM_true(vthreadInspectorCount > 0);
-	while (!objectAccessBarrier.inlineMixedObjectCompareAndSwapU64(
-														currentThread,
-														threadObj,
-														vm->virtualThreadInspectorCountOffset,
-														(U_64)vthreadInspectorCount,
-														((U_64)(vthreadInspectorCount - 1)))
-	) {
-		/* Field updated by another thread, try again. */
-		vthreadInspectorCount = J9OBJECT_I64_LOAD(currentThread, threadObj, vm->virtualThreadInspectorCountOffset);
+	U_64 isSuspendedFlag = J9OBJECT_U64_LOAD(currentThread, threadObj, vm->isSuspendedInternalOffset);
```

Comment:
`isSuspendedFlag` or `isSuspendedInternalOffset`: it is no longer a boolean; so, these names should be updated to reflect the new behaviour.